src/internal/operators/last.ts

```diff
@@ -29,6 +29,38 @@ export function last<T, D = T>(
  * from the source Observable that satisfies the predicate.
  *
  * ![](last.png)
+ * 
+ * It will throw an error if the notification is sent but the source is completed. It returns the
```

Comment:
I think this should be a little bit clearer, e.g. 
"It will throw an error if the source completes without notification or one that matches the predicate"

Reply:
Just updated.